about-us/index.html

```diff
@@ -82,16 +82,6 @@ <h2 class="title-borderbottom">OUR Values</h2>
             <li>We're <b>Inclusive</b>. We believe that anyone can learn to help others through art, no matter their skill set!</li>
             <li>We're <b>Passionate</b>. We’re committed to what we do and want to share our love for creating crafts.</li>
         </ul>
-        
-        <h2 class="title-borderbottom">PROGRESS</h2>
```

Comment:
Former commit in here - change to master branch before checking out a new branch.